base/linalg/generic.jl

```diff
@@ -712,7 +712,7 @@ function issymmetric(A::AbstractMatrix)
     return true
 end
 
-issymmetric(x::Number) = true
+issymmetric(x::Number) = x == x
```

Comment:
I thought about that but I actually like `x==x` because it's the definition of symmetry collapsed to a scalar.